atd-vzd/migrations/default/1697836965147_add_units_change_log_function/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE FUNCTION public.units_change_log_insert ()
+	RETURNS TRIGGER
+	LANGUAGE plpgsql
+	AS $function$
+BEGIN
+	-- Insert the previous record values from the unified public.atd_txdot_units view into the change log table
+	INSERT INTO public.atd_txdot_change_log (record_id, record_crash_id, record_type, record_json)
+		VALUES(old.unit_id, old.crash_id, 'units', (
+				SELECT
+					row_to_json(atd_txdot_units)
+					FROM public.atd_txdot_units
+				WHERE
+					unit_nbr = old.unit_nbr
+					AND crash_id = old.crash_id));
+	RETURN NEW;
+END;
+$function$;
+
+CREATE TRIGGER units_change_log_insert
+	BEFORE UPDATE ON vz_facts.atd_txdot_units
+	FOR EACH ROW
+	EXECUTE FUNCTION public.units_change_log_insert ();

```

Comment:
this looks good, also just had me thinking back to our convo in the LDM ideation meeting about whether CRIS updates are something we would want to track in change logs at some point? i think the consensus was yes but might be good to talk to the VZ team to make sure thats something they want/would use.